addons/project/models/project.py

```diff
@@ -11,6 +11,7 @@
 from odoo import api, Command, fields, models, tools, SUPERUSER_ID, _
 from odoo.exceptions import UserError, ValidationError, AccessError
 from odoo.tools import format_amount
+from odoo.osv import expression
 from odoo.osv.expression import OR
```

Comment:
Can you remove this import since your import have the OR function too?
And can you update this line to use `expression.OR` instead. 😇 
Thanks in advance! 🙂

Reply:
Do you mean removing this import and importing AND as well on the following line? I need to get access to AND somehow as I use it afterwards.

Reply:
No, in fact I said you can remove the import `from odoo.osv.expression import OR` and change the using of `OR` by `expression.OR` in this file. 😇
But if you have removed your import by adding `AND` in the one below, it is okay too. 😇